process/outportBlockConverter.go

```diff
@@ -0,0 +1,173 @@
+package process
+
+import (
+	"fmt"
+	"math/big"
+
+	"github.com/multiversx/mx-chain-core-go/core"
+	coreData "github.com/multiversx/mx-chain-core-go/data"
+	"github.com/multiversx/mx-chain-core-go/data/block"
+	"github.com/multiversx/mx-chain-core-go/data/outport"
+	"github.com/multiversx/mx-chain-core-go/marshal"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/multiversx/mx-chain-ws-connector-template-go/data"
+)
+
+type outportBlockConverter struct {
+	gogoProtoMarshalizer *marshal.GogoProtoMarshalizer
+	bigIntCaster         coreData.BigIntCaster
```

Comment:
we can have here a separate component for standard proto marshaller, which implements marshal.Marshaller from core